### PR TITLE
i18n: adds Occitan locale

### PR DESCRIPTION
**Pull request by Mejans**

### CODE REVIEW

**Comment by robert-ancell** [comment]
@Mejans the build is failing, can you check?

**Comment by Mejans** [comment]
Hello @robert-ancell 
I usually just translate this way and it woks. I don't understand the report neither what it complains about =/

**Comment by robert-ancell** [comment]
@sergio-costas you added the gettext support - can you see what is wrong with the PR?

**Comment by sergio-costas** [comment]
@robert-ancell You added the oc.po file in the root folder. It must be inside the po folder.

File: oc.po [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+"Language-Team: \n"
+"Language: oc\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/main.c:59 src/main.c:65 src/main.c:88
+msgid "Installing missing theme snaps:"
+msgstr "Installacion de snaps de tèma mancants :"
```
Comment (sergio-costas) [inline oc.po:22]: That blank space between the colon... Is a typo?

Comment (Mejans) [reply oc.po:22]: no, it's like in French, we put a no-break space before :  ! and ?

**Comment by Mejans** [comment]
> @robert-ancell You added the oc.po file in the root folder. It must be inside the po folder.

I have just edited the file so it's in the propper folder!

**Pull request merged**